src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -793,6 +794,12 @@ ActStatus ActionHandlers::process_cs_resp(SM::ControlBlock& cb)
 	const gtp_incoming_msg_data_t* gtp_msg_data= static_cast<const gtp_incoming_msg_data_t*>(msgBuf->getDataPointer());
 	const struct csr_Q_msg& csr_info = gtp_msg_data->msg_data.csr_Q_msg_m;
 
+    if(csr_info.status != GTPV2C_CAUSE_REQUEST_ACCEPTED)
```

Comment:
You had mentioned that there was a mme-app crash. What was the root cause of the crash? Is there a NULL check missing or boundary condition that we are not handling?